bin/build-manylinux.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+# This script builds BlazingMQ and all of its dependencies.
+#
+# Before running this script, install following prerequisites, if not present
+# yet, by copy-and-pasting the commands between `<<PREREQUISITES` and
+# `PREREQUISITES` below:
+<<PREREQUISITES
+sudo yum install -y \
+    gdb \
+    curl \
+    pkgconfig \
+    ninja-build \
+    flex-devel \
+    zlib-devel \
+    m4
+PREREQUISITES
+                                                    # shellcheck disable=SC2188
```

Comment:
This line should go before the `<<PREREQUISITES` to stop the shellcheck warning.